dparc.tex

```diff
@@ -26,10 +26,10 @@ \subsection{Prompt \gls{Enclave}} \label{sec:promptenc}
 The alerts can be further subdivided into \emph{streak} alerts for objects that have moved a long distance and \emph{non-streak} alerts for all other objects.
 Measurements in the catalogs follow the same subdivision. Images may be \gls{Commissioning} images used for testing and characterizing the Observatory systems, normal science images without significant \emph{streaks}, or delayed science images that do contain significant \emph{streaks}.
 
-\emph{Streak} alerts are transmitted to an \gls{Alert} Vetting System (AVS) located at a Trusted Broker Facility at  Lawrence Livermore National Lab.
-\emph{Non-streak} alerts and \emph{streak} alerts approved by the \gls{AVS} are to be published to the world at large within 60 seconds of the original raw image being taken.
+\emph{Streak} alerts are checked against the \gls{OUO} catalog at SLAC before release.
+\emph{Non-streak} alerts and \emph{streak} alerts agreed with \gls{JPL} \gls{NEO} group  are to be published to the world at large within 60 seconds of the original raw image being taken.
```

Comment:
The JPL-vetted streaks will have to be released on timescales longer than 60 seconds--they won't be alerts, but might be transmitted to MPC as part of solar system processing.